depenguinme.sh

```diff
@@ -158,6 +158,12 @@ while [ "$#" -gt 0 ]; do
 	shift
 done
 
+DEPS=(
```

Comment:
Having DEPS here makes sense, since it's not really a configuration option (compared to the other settings above, like DEFAULT_QEMU_RAM) :+1: